docs/repositories-configure/code-patterns.md

```diff
@@ -14,14 +14,14 @@ However, you can create a coding standard adapted to your scenario by configurin
 
 1.  Enable or disable the patterns using the checkbox for each pattern.
 
-    To see a detailed explanation about the issues that a pattern can detect and how to fix them, click **Details** on the top right-hand corner of the pattern. Some patterns also allow you to configure the rules:
+    To see an explanation of the issues that a pattern detects and how to fix them, click **Show details**. Some patterns also allow you to configure the rules for detecting issues:
```

Comment:
It is weird that you are highlighting the action "show details" but in the screenshot you highlight "hide details"

Reply:
just a detail, I will approve anyway

Reply:
Yes, before the label didn't change when the details were expanded:

![image](https://user-images.githubusercontent.com/60105800/127651031-9eb886f4-0ea3-45a8-88fe-f32fc7aa45d1.png)

The main purpose of this screenshot is actually to show the expanded view of the code pattern. So what I'll do is remove the highlight from the "Hide details", as it isn't that important anyways. :ok_hand:

Reply:
Coool